dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/WebSphereRuntimeComponentProvider.java

```diff
@@ -71,13 +71,16 @@
                 }
             }
 
-            // Check if EAR 7.0 applications are supported by the runtime
+            // Check what version of EAR applications are supported by the runtime
             if (RuntimeManager.isRuntimeComponentTypeDefined("com.ibm.ws.st.runtime.enterpriseApplication")) {
                 IRuntimeComponentType rct = RuntimeManager.getRuntimeComponentType("com.ibm.ws.st.runtime.enterpriseApplication");
                 if (rct != null) {
                     if (wr.isEARSupported("7.0") && rct.hasVersion("7.0")) {
                         list.add(RuntimeManager.createRuntimeComponent(rct.getVersion("7.0"), new HashMap<String, String>(0)));
                     }
+                    if (wr.isEARSupported("8.0") && rct.hasVersion("8.0")) {
```

Comment:
Should the check for 8.0 come before 7.0?

Reply:
It adds both to the list if available.  Same as the code above where it adds everything to the list that is available.